lms/static/js/fixtures/edxnotes/edxnotes.html

```diff
@@ -0,0 +1,28 @@
+<div class="wrapper-student-notes">
+    <div class="title-search-container">
+        <div class="wrapper-title">
+            <h1 class="page-title">
+                Notes
+                <small class="page-subtitle">Highlights and notes you've made in course content</small>
+            </h1>
+        </div>
+        <div class="wrapper-notes-search">
+            <form role="search" action="/search_endpoint" method="GET" id="search-notes-form" class="is-hidden">
+                <label for="search-notes-input" class="sr">Search notes for:</label>
+                <input type="search" class="search-notes-input" id="search-notes-input" name="note" placeholder="Search notes for...">
+                <button type="submit" class="search-notes-submit"><i class="icon-search"></i> Search</button>
+            </form>
+        </div>
+    </div>
+
+    <section class="wrapper-tabs">
+        <div class="tab-list is-hidden">
+            <h2 id="tab-view" class="tabs-label">View notes by:</h2>
+        </div>
+        <div class="inline-error is-hidden" tabindex="-1"></div>
+        <div class="ui-loading" tabindex="-1">
+            <p><span class="spin"><i class="icon-refresh"></i></span> <span class="copy">Loading...</span></p>
```

Comment:
@andy-armstrong 
> Please remove the '...' from the Loading message as we have decided to no longer add ellipses.

Done, removed it from:

`lms/templates/edxnotes/edxnotes.html`

and

`lms/static/js/fixtures/edxnotes/edxnotes.html`